js/lightbeam.js

```diff
@@ -113,6 +113,16 @@ async function initLightBeam() {
   websites = await storeChild.getAll();
   renderGraph(websites);
 
+  const saveData = document.getElementById('save-data-button');
+  saveData.addEventListener('click', () => {
+    storeChild.getAll().then( data => {
```

Comment:
Can we use await here instead of then?

Either use a named function callback to the event listener or change it to `'click', async function () {`

Can we look for the ESLint rule that requires consistent spacing too? So far we have functions with arguments that have spaces between but not spaces before `.then(data => {` rather than `.then( data =>`